### ISSUE TITLE
Low-contrast in light mode makes text unreadable

### ISSUE THREAD

**Issue by focusaurus** [chipsenkbeil/choose#20]
Choose seems to pick dark mode or light mode based on what's behind it on the screen (I guess?). Dark mode is OK, but light mode has low contrast between text and background for the non-selected items and I can barely read them.

I see there's a new 1.1 release but it's not on homebrew yet it seems so I can't easily test the new CLI options yet. I'm using 1.0 currently.

![Screenshot on 2020-06-05 at 09-22-15](https://user-images.githubusercontent.com/482377/83894192-33e13780-a70e-11ea-997e-dcdd05b5e408.png)

**Comment by focusaurus** [comment]
Note with 1.1 I still have this issue.

Also I submitted a [pull request to homebrew-core](https://github.com/Homebrew/homebrew-core/pull/55845) to switch to the chipsenkbeil account and update to 1.1.

**Comment by chipsenkbeil** [comment]
Beyond 1.1, in the work towards 1.2, I've started implementing different requests from the issue list. The first was changing some of the colors to better support dark and light modes. This landed with https://github.com/chipsenkbeil/choose/commit/66e0f51869aa0e936bceb952854f8df502193639. Sounds like you haven't been able to try it yet to see if it resolves your issue.

Could we update the homebrew formula to support building from HEAD? I know that was a feature that used to exist in homebrew, but don't know if it's still there. That way, you could build more easily and try out new features that don't yet have a tag.

Thoughts?

**Comment by focusaurus** [comment]
Now that I got my fork etc set up for homebrew-core work, I should be able
to do a build/install of HEAD to test stuff. For some reason choose-gui
builds fine in homebrew but if I try building directly with `make` it
doesn't work and I'm not familiar with macos native dev at all so not
really able to troubleshoot.

I think the actual merged formula should remain the latest official release
as is normal.

**Comment by focusaurus** [comment]
It looks like I can get a debug build to work locally but not a release build. This issue is a pretty significant usability struggle for me. Do you have time and inclination to just release a patch or minor release with 66e0f51?

**Comment by chipsenkbeil** [comment]
Sure, just tagged a new release for latest commit. https://github.com/chipsenkbeil/choose/releases/tag/1.2

It might also be good to support building head using homebrew `brew install choose-gui --HEAD`. I thought formula supported this out of the box if you specify the repo, but looks like you have to provide a *head* parameter: https://docs.brew.sh/Formula-Cookbook#unstable-versions-head

**Comment by focusaurus** [comment]
I agree on adding a `head` line to the formula. I tried to do that but don't want to PR into homebrew until it's actually succeeding for me and I'm still hitting the lack of release build missing file issue.

Looking at your homebrew PR, it's failing CI. I'm thinking `choose -h` is either returning nonzero or maybe something is hanging and timing out?

**Comment by chipsenkbeil** [comment]
@focusaurus, with cdc986e2746194f21857d37b5a771f42652cd2d5, I changed the default build profile to debug. You can build the release version with `xcodebuild -configuration Release build`. Have you been getting issues with that configuration? Separately, not sure why the PR is failing at the moment.

**Comment by focusaurus** [comment]
I did a release build but the executable fails with `illegal hardware instruction` so I'll just wait for that homebrew PR to land.

**Comment by focusaurus** [comment]
@chipsenkbeil How do you think we should try to help the homebrew-core PR land? I'm not sure why `choose -h` seems to be timing out in CI for them. Locally it works and exits code `0` for me.

**Comment by chipsenkbeil** [comment]
@focusaurus, I'm not sure at this time. My availability to dive into homebrew formulas is a bit limited at the moment, so if anyone else wants to investigate I'd be happy to have them take over the PR for homebrew (and any changes we need to make here).

**Comment by chipsenkbeil** [comment]
@focusaurus version 1.2.1 has been merged into homebrew formula. Give it a try and see if it helps.

**Comment by focusaurus** [comment]
Perfecto! Thanks so much. Also shout out to my coworker @garretcarrot for that bug fix which unblocked this!
